scripts/tests/UITests/alice-user-login_test.go

```diff
@@ -0,0 +1,35 @@
+package UITests
```

Comment:
use standard naming for both the package and the script names and copyright header is missing.